doc/_extensions/zephyr/kconfig/__init__.py

```diff
@@ -62,13 +63,13 @@
 KCONFIGLIB = SCRIPTS / "kconfig"
 sys.path.insert(0, str(KCONFIGLIB))
 
-import kconfiglib
-import list_boards
-import list_hardware
-import zephyr_module
+import kconfiglib  # noqa: E402
+import list_boards  # noqa: E402
+import list_hardware  # noqa: E402
+import zephyr_module  # noqa: E402
```

Comment:
Same here

Reply:
same here :) it's the fact that there is more than just sys.path.insert() calls that's causing the check to not pass